src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -96,6 +97,15 @@ export class BaseEditCourseForm extends React.Component {
     const languageOptions = (courseRunOptions &&
       this.parseOptions(courseRunOptions.content_language.choices));
 
+    let submitState = 'default';
+    if (submitting) {
+      submitState = 'pending';
+    } else if (pristine) {
+      // FIXME: Once this form is correctly reset to a pristine state after a successful submit,
```

Comment:
We will need to manually figure out how to do this given the `keepDirtyOnReinitialize` form parameter defined at the bottom of this file.